compiler/yarn.lock

```diff
@@ -113,6 +113,7 @@ ajv@^6.5.5:
 "alan-js-runtime@file:../js-runtime":
   version "0.0.1"
   dependencies:
+    isomorphic-fetch "^2.2.1"
```

Comment:
shouldn't this be a `binary` file in `.gitattributes`?

Reply:
Yeah, probably. :) I just always forget that sort of thing.